### PR TITLE
[media-video/ffmpegthumbnailer] Version bump

### PR DESCRIPTION
**Pull request by a17r**
Project switched to cmake and github
See also bugs 517922, 549422

### CODE REVIEW

File: media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+inherit cmake-utils
+
+DESCRIPTION="Lightweight video thumbnailer that can be used by file managers"
+HOMEPAGE="https://github.com/dirkvdb/ffmpegthumbnailer"
+SRC_URI="https://github.com/dirkvdb/${PN}/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
```
Comment (mgorny) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: Why are you removing all the keywords?

Comment (a17r) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: oh, I can only test on that one. but they are easily restored

File: media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+SRC_URI="https://github.com/dirkvdb/${PN}/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE="gnome gtk jpeg libav png test"
+
+RDEPEND="
+	gtk? ( >=dev-libs/glib-2.30 )
+	jpeg? ( virtual/jpeg:0 )
```
Comment (mgorny) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: Why not pin the above two via :0= / :2=?

File: media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+src_prepare() {
+	rm -rf "${S}"/out* || die
+
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		$(cmake-utils_use png HAVE_PNG)
+		$(cmake-utils_use jpeg HAVE_JPEG)
```
Comment (mgorny) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: Could you switch all the way to usex? It just looks bad to mix two syntaxes for the same goal.

**Comment by a17r** [comment]
Added another upstream patch to fix and un-restrict tests ;)

File: media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+SRC_URI="https://github.com/dirkvdb/${PN}/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE="gnome gtk jpeg libav png test"
+
+RDEPEND="
+	gtk? ( dev-libs/glib:2 )
+	jpeg? ( virtual/jpeg:0 )
```
Comment (mgorny) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: Didn't I ask already why not `:2=` and `:0=`?

Comment (a17r) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: Indeed you have, and I overlooked it

File: media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+PATCHES=(
+	"${FILESDIR}/${P}-config-summary.patch"
+	"${FILESDIR}/${P}-installdirs.patch"
+	"${FILESDIR}/${P}-set-locale.patch"
+)
+
+DOCS=( AUTHORS ChangeLog README.md )
+
+src_prepare() {
+	rm -rf "${S}"/out* || die
```
Comment (mgorny) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: ${S} is redundant here, src_prepare() starts in ${S}.

Comment (a17r) [inline media-video/ffmpegthumbnailer/ffmpegthumbnailer-2.0.10.ebuild]: indeed

**Comment by mgorny** [comment]
```
+*ffmpegthumbnailer-2.0.10 (26 Jul 2015)
+
+  26 Jul 2015; Michał Górny <mgorny@gentoo.org>
+  +ffmpegthumbnailer-2.0.10.ebuild,
+  +files/ffmpegthumbnailer-2.0.10-config-summary.patch,
+  +files/ffmpegthumbnailer-2.0.10-installdirs.patch,
+  +files/ffmpegthumbnailer-2.0.10-set-locale.patch:
+  Version bump. Update homepage and switch to cmake. https://github.com/gentoo
+  /gentoo-portage-rsync-mirror/pull/175 by a17r.
```

**Pull request closed without merging**